### ISSUE TITLE
Can't train openface face recognition model

### ISSUE THREAD

**Issue by iamBansiG** [cmusatyalab/openface#425]
I tried to train a new classifier for my dataset. While training the classifier i received following error. How to solve this error ?

{
  retrain : "none"
  imagesPerPerson : 3
  save : "/home/ubuntu/openface/models/openface"
  nEpochs : 3
  epochNumber : 1
  cudnn : true
  epochSize : 2
  cudnn_bench : false
  nGPU : 1
  testing : true
  embSize : 128
  device : 1
  cuda : false
  alpha : 0.2
  modelDef : "../models/openface/nn4.small2.def.lua"
  data : "/home/ubuntu/openface/data/mydataset"
  peoplePerBatch : 2
  lfwDir : "../data/aligneddata"
  imgDim : 96
  testBatchSize : 8
  manualSeed : 2
  cache : "/home/ubuntu/openface/training/work"
  nDonkeys : 2
}
Saving everything to: /home/ubuntu/openface/models/openface	
Starting donkey with id: 1 seed: 3
Starting donkey with id: 2 seed: 4
Loading train metadata from cache
Loading train metadata from cache
nClasses: 	3	
==> doing epoch on training data:	
==> online epoch # 1	
  + (nTrips, nTripsFound) = (6, 6)	
Epoch: [1][1/2]	Time 8.661	tripErr 2.17e-01	
  + (nTrips, nTripsFound) = (6, 4)	
Epoch: [1][2/2]	Time 8.799	tripErr 3.70e-01	
Epoch: [1][TRAINING SUMMARY] Total Time(s): 17.81	average triplet loss (per batch): 0.29	

	
/home/ubuntu/torch/install/bin/lua: /home/ubuntu/openface/training/train.lua:104: bad argument #3 to 'format' (string expected, got table)
stack traceback:
	[C]: in function 'format'
	/home/ubuntu/openface/training/train.lua:104: in function 'saveModel'
	./main.lua:45: in main chunk
	[C]: in function 'dofile'
	.../torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
	[C]: ?
